todo-app/src/store/TodoStore.js

```diff
@@ -0,0 +1,126 @@
+import { observable, action, useStrict, runInAction } from "mobx";
+
+const BASE_URL = "https://hyf-react-api.herokuapp.com/todos/";
+
+useStrict(true);
+// will throw an error if you didn't use the @action or didn't use any decorator
+
+export default class TodoStore {
+  @observable todos = [];
+
+  @observable loading = false;
+
+  @observable isEditing = null;
```

Comment:
This is now confusing. You have `loading = false`, and then immediately below you have `isEditing = null`. To be more accurate, and to make sense to future devs reading the code, you should probably change this to something like `isEditingID`.